rb/lib/selenium/webdriver/remote/w3c/capabilities.rb

```diff
@@ -0,0 +1,253 @@
+# encoding: utf-8
+#
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+module Selenium
+  module WebDriver
+    module Remote
+      module W3C
+        #
+        # Specification of the desired and/or actual capabilities of the browser that the
+        # server is being asked to create.
+        #
+
+        # TODO - uncomment when Mozilla fixes this:
+        # https://bugzilla.mozilla.org/show_bug.cgi?id=1326397
+        class Capabilities
+
+          # TODO (alex): compare with spec
+          KNOWN = [
+            :browser_name,
+            :browser_version,
+            :platform_name,
+            :platform_version,
+            :accept_insecure_certs,
+            :page_load_strategy,
+            :proxy,
+            :remote_session_id,
+            :accessibility_checks,
+            :device,
+            :implicit_timeout,
+            :page_load_timeout,
+            :script_timeout,
+          ].freeze
```

Comment:
Spec has required capabilities of:

- browser_name
- browser_version
- platform_name
- accept_insecure_certs
- page_load_strategy
- proxy
- set_window_rect
- timeouts
- unhandled_prompt_behavior